pkg/server/tenant.go

```diff
@@ -787,7 +787,7 @@ func (s *SQLServerWrapper) PreStart(ctx context.Context) error {
 		s.sqlCfg.ExternalIODirConfig,
 		s.sqlServer.cfg.Settings,
 		s.sqlServer.sqlIDContainer,
-		s.nodeDialer,
+		s.kvNodeDialer, // TODO(ssd): is this right?
```

Comment:
Yes, I believe it is.  This is used in the external storage layer for the nodelocal service. In 23.1 we made nodelocal more clearly a KV-node service gated by a capability to use.

Reply:
Ok, thanks. I removed the comment